browser-specific-failures.js

```diff
@@ -0,0 +1,386 @@
+'use strict';
+
+/**
+ * Implements a view of how many browser specific failures each engine has over
+ * time.
+ */
+
+const fetch = require ('node-fetch');
+const fs = require ('fs');
+const flags = require('flags');
+const Git = require('nodegit');
+const lib = require('./lib');
+const moment = require('moment');
+
+flags.defineString('from', '2018-07-01', 'Starting date (inclusive)');
+flags.defineString('to', moment().format('YYYY-MM-DD'), 'Ending date (exclusive)');
+flags.defineStringList('products', ['chrome', 'firefox', 'safari'], 'Browsers to compare. Must match the products used on wpt.fyi');
+flags.defineString('output', null, 'Output CSV file to write to. Defaults to {stable, experimental}-browser-specific-failures.csv');
+flags.defineBoolean('experimental', false, 'Calculate metrics for experimental runs.');
+flags.parse();
+
+// Helper function that counts how many test failures there are within a given
+// set of status results.
+function countFailures(statuses) {
+  let count = 0;
```

Comment:
Or `return statuses.filter(s => s == 'FAIL').length`. Not too clever IMO, but you decide :)